app/services/true_layer/api_client.rb

```diff
@@ -57,7 +57,7 @@ def simple_error(error)
 
     def connection
       @connection ||= Faraday.new(url: TRUE_LAYER_URL) do |conn|
-        conn.authorization :Bearer, token
+        conn.request :authorization, 'Bearer', token
```

Comment:
You don't need this change as this is being fixed by Colin's PR.

Reply:
decided with Colin to put all the commits into this one PR

Reply:
👍